From ef603e4d4f62add4191e893dfa3bc79cf6bb35a7 Mon Sep 17 00:00:00 2001
From: Huynh Thanh Hung <hung.huynh,wz@renesas.com>
Date: Tue, 28 May 2019 11:19:28 +0700
Subject: [PATCH 154/215] arm64: dts: renesas: hihope-common: enable usb3_peri0

This patch enable usb3_peri0 that usb usb3_phy0 to enable VBUS
detection for the USB3.0 peripheral.

Signed-off-by: Huynh Thanh Hung <hung.huynh,wz@renesas.com>
---
 arch/arm64/boot/dts/renesas/hihope-common.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/hihope-common.dtsi b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
index 86d5403..36791c24 100644
--- a/arch/arm64/boot/dts/renesas/hihope-common.dtsi
+++ b/arch/arm64/boot/dts/renesas/hihope-common.dtsi
@@ -288,6 +288,13 @@
 	status = "okay";
 };
 
+&usb3_peri0 {
+	phys = <&usb3_phy0>;
+	phy-names = "usb";
+
+	status = "okay";
+};
+
 &usb3_phy0 {
 	status = "okay";
 };
-- 
2.7.4

